src/main/java/frc/team2412/robot/subsystem/DrivebaseSubsystem.java

```diff
@@ -94,17 +105,23 @@
     private final NetworkTableEntry odometryXEntry;
     private final NetworkTableEntry odometryYEntry;
     private final NetworkTableEntry odometryAngleEntry;
+    private final NetworkTableEntry module1, module2, module3, module4;
     private final NetworkTableEntry isFieldOrientedEntry;
 
-    public DrivebaseSubsystem(SwerveModule fl, SwerveModule fr, SwerveModule bl, SwerveModule br, Gyroscope g) {
+    private final Field2d field = new Field2d();
```

Comment:
Hrm.  Robot.java is also creating a Field2d.  I'd recommend assigning that one here instead of creating a new one, with ` field = Robot.getInstance().field`

Reply:
Robot.getInstance is null, it causes a crash from nullpointer exception

Reply:
Yep - you need to move the creation of the Field2D before the subsystems.  See the comment in Robot.java